src/controllers/update/overseas.entity.review.controller.ts

```diff
@@ -15,6 +18,30 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     const changeLinkUrl: string = config.OVERSEAS_ENTITY_UPDATE_DETAILS_URL;
     const overseasEntityHeading: string = "Check the overseas entity details";
 
+    if (!isActiveFeature(config.FEATURE_FLAG_DISABLE_UPDATE_PRIVATE_DATA_FETCH)) {
+
+      // Fetch OE email address if not already in session.
+      const overseasEntityId = appData.overseas_entity_id;
+      const transactionId = appData.transaction_id;
+      if (!appData.entity?.email && overseasEntityId !== undefined && transactionId !== undefined) {
```

Comment:
Is it just undefined we check for here? Are they allowed to be null etc? Maybe:

if (!(appData?.entity?.email) && overseasEntityId && transactionId)

Reply:
It's just to get over the "string | undefined" in the typing. So do not expect them to be null.